CONTRIBUTING.md

```diff
@@ -28,45 +30,54 @@ Please reserve GitHub issues for feature requests and bugs rather than general q
 
 Read the usage and testing steps in the [README](README.md).
 
-
 ## Contributing
 
 ### Report a Bug
 
-To report a bug, open an issue on GitHub with the label `bug` using the available bug report issue template. Please ensure the issue has not already been reported.
+To report a bug, open an issue on GitHub with the label `bug` using the available bug report issue template. Please
+ensure the issue has not already been reported.
 
 ### Suggest an Enhancement
 
-To suggest an enhancement, please create an issue on GitHub with the label `enhancement` using the available feature issue template.
+To suggest an enhancement, please create an issue on GitHub with the label `enhancement` using the available feature
+issue template.
 
 ### Open a Pull Request
 
-* Fork the repo, create a branch, submit a PR when your changes are tested and ready for review
-* Fill in [our pull request template](.github/PULL_REQUEST_TEMPLATE.md)
+- Fork the repo, create a branch, submit a PR when your changes are tested and ready for review
+- Fill in [our pull request template](.github/PULL_REQUEST_TEMPLATE.md)
 
 > **Note**
 >
-> If you’d like to implement a new feature, please consider creating a feature request issue first to start a discussion about the feature.
+> If you’d like to implement a new feature, please consider creating a feature request issue first to start a discussion
+> about the feature.
```

Comment:
```suggestion
> If you’d like to implement a new feature, start by creating a feature request issue to start a discussion
> about the feature.
```
Suggested re-wording. Let me know what you think.

Reply:
This should technically follow the same governance structure that we have in place with NIC and NKG.
However, this PR is not the time to address that.

Reply:
I will update the guide in another PR 👍